src/main/webapp/editfile.html

```diff
@@ -27,7 +27,7 @@ <h2>Edit Configuration</h2>
         <label for="account-id">Account ID:</label>
         <select id="account-ids">
         </select><br><br>
-        <button type="submit">Search</button>
+        <button type="submit"onclick="javascript:populateEditForm()">Edit</button>
```

Comment:
does this interfere with the form's onsubmit attribute?

Reply:
No, the edit form will only be shown after the submit button is reached. If there is a problem during that process(a missing field or something), the populateEditForm() will be irrelevant because they will be no form to fill.